gulpfile.js

```diff
@@ -117,6 +134,16 @@ function sniffs(done) {
     
 }
 
+// run webpack
+gulp.task('webpack', webpack);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['webpack' was used before it was defined. (no-use-before-define)](https://app.codacy.com/app/uriweb/uri-component-library/pullRequest?prid=4632651)